packages/applications-service-api/prisma/schema.prisma

```diff
@@ -74,6 +74,24 @@ model Representation {
   @@index([caseReference])
 }
 
+model ServiceUser {
+  id               Int      @unique @default(autoincrement())
+  serviceUserId    Int      @id
+  caseReference    String?
+  firstName        String?
+  middleName       String?
+  lastName         String?
+  email            String?
+  webAddress       String?
```

Comment:
this is named `website` in the Back Office db. Should we keep it consistent across BO db -> message -> FO db?

Reply:
For all of these I've based it off the message which isn't consistent with BO DB.

Perhaps I'll change to be consistent with BO DB.